src/controllers/simplified-account/services/payment-links/create/create-payment-link.controller.ts

```diff
@@ -1,18 +1,202 @@
 import { ServiceRequest, ServiceResponse } from '@utils/types/express'
+import { response } from '@utils/response'
+import { body, validationResult } from 'express-validator'
+import formatValidationErrors from '@utils/simplified-account/format/format-validation-errors'
+import formatServiceAndAccountPathsFor from '@utils/simplified-account/format/format-service-and-account-paths-for'
+import paths from '@root/paths'
+import { NextFunction } from 'express'
+import formatAccountPathsFor from '@utils/format-account-paths-for'
+import { PublicAuthClient } from '@services/clients/pay/PublicAuthClient.class'
+import { CreateTokenRequest } from '@models/public-auth/CreateTokenRequest.class'
 
-function get (req: ServiceRequest, res: ServiceResponse) {
-  return res.status(501).json({
-    message: 'not implemented'
-  })
+// @ts-expect-error: Missing type definitions for @govuk-pay/pay-js-commons module
+import payJsCommons from '@govuk-pay/pay-js-commons'
+
+const { slugify, removeIndefiniteArticles } = (payJsCommons as unknown as {
+  nunjucksFilters: {
+    slugify: (input: string) => string
+    removeIndefiniteArticles: (input: string) => string
+  }
+}).nunjucksFilters
+
+const getLanguageFromQuery = (req: ServiceRequest): string => {
+  const queryLanguage = req.query?.language
+  return typeof queryLanguage === 'string' ? queryLanguage : ''
+}
+
+interface Service {
```

Comment:
this isn't needed, we already have a Service class that can be used as a type deff

Reply:
Thanks, amending now. (Happy that it's already a green branch, thanks!)